fix(Code): Circular dependency error in Swagger Module

## 💡 Details

Deployment failed due to circular dependency error in Swagger Module.

```
Run npm run docs:openapi

> server@0.0.1 docs:openapi
> npx tsx infra/scripts/generate-openapi.ts

npm warn exec The following package was not found and will be installed: tsx@4.19.3
[Nest] 2192  - 04/16/2025, 12:37:13 PM     LOG [NestFactory] Starting Nest application...
[Nest] 2192  - 04/16/2025, 12:37:13 PM     LOG [InstanceLoader] AppModule dependencies initialized +9ms
[Nest] 2192  - 04/16/2025, 12:37:13 PM     LOG [InstanceLoader] DockerfilesModule dependencies initialized +0ms
[Nest] 2192  - 04/16/2025, 12:37:13 PM     LOG [InstanceLoader] ConfigHostModule dependencies initialized +0ms
[Nest] 2192  - 04/16/2025, 12:37:13 PM     LOG [InstanceLoader] ConfigModule dependencies initialized +0ms
❌ Failed to generate OpenAPI spec: Error: A circular dependency has been detected (property key: "id"). Please, make sure that each side of a bidirectional relationships are using lazy resolvers ("type: () => ClassType").
    at SchemaObjectFactory.createNotBuiltInTypeReference (/home/runner/work/docker-tutorial/docker-tutorial/server/node_modules/@nestjs/swagger/dist/services/schema-object-factory.js:259:19)
    at SchemaObjectFactory.createSchemaMetadata (/home/runner/work/docker-tutorial/docker-tutorial/server/node_modules/@nestjs/swagger/dist/services/schema-object-factory.js:376:25)
    at SchemaObjectFactory.mergePropertyWithMetadata (/home/runner/work/docker-tutorial/docker-tutorial/server/node_modules/@nestjs/swagger/dist/services/schema-object-factory.js:187:21)
    at /home/runner/work/docker-tutorial/docker-tutorial/server/node_modules/@nestjs/swagger/dist/services/schema-object-factory.js:115:35
    at Array.map (<anonymous>)
    at SchemaObjectFactory.extractPropertiesFromType (/home/runner/work/docker-tutorial/docker-tutorial/server/node_modules/@nestjs/swagger/dist/services/schema-object-factory.js:114:52)
    at SchemaObjectFactory.exploreModelSchema (/home/runner/work/docker-tutorial/docker-tutorial/server/node_modules/@nestjs/swagger/dist/services/schema-object-factory.js:139:41)
    at ResponseObjectFactory.create (/home/runner/work/docker-tutorial/docker-tutorial/server/node_modules/@nestjs/swagger/dist/services/response-object-factory.js:49:47)
    at /home/runner/work/docker-tutorial/docker-tutorial/server/node_modules/@nestjs/swagger/dist/explorers/api-response.explorer.js:67:101
    at /home/runner/work/docker-tutorial/docker-tutorial/server/node_modules/lodash/lodash.js:13469:38
Error: Process completed with exit code 1.
```

## References

- https://github.com/docker-seminar/docker-tutorial/actions/runs/14492871057
- https://github.com/nestjs/swagger/issues/1475

--- a/server/src/dockerfiles/dto/dockerfile-base.response.dto.ts
+++ b/server/src/dockerfiles/dto/dockerfile-base.response.dto.ts
@@ -12,6 +12,9 @@ export abstract class DockerfileBaseResponseDto {
 	 */
 	id: string
 
+	@ApiProperty({
+		enum: DockerfileStatus,
+	})
 	/**
 	 * Current processing state of the Dockerfile.
 	 */

